services/director-v2/src/simcore_service_director_v2/modules/remote_debug.py

```diff
@@ -0,0 +1,32 @@
+""" Setup remote debugger with Python Tools for Visual Studio (PTVSD)
+
+"""
+import logging
+
+from fastapi import FastAPI
+
+logger = logging.getLogger(__name__)
+
+REMOTE_DEBUG_PORT = 3000
+
+
+def setup(app: FastAPI):
+    def on_startup() -> None:
+        try:
+            logger.debug("Enabling attach ptvsd ...")
+            #
+            # SEE https://github.com/microsoft/ptvsd#enabling-debugging
+            #
+            import ptvsd
+
+            ptvsd.enable_attach(
+                address=("0.0.0.0", REMOTE_DEBUG_PORT),  # nosec
+            )  # nosec
+        except ImportError as err:
+            raise ValueError(
```

Comment:
Maybe raising just `Exception` makes more sense.

Reply:
Done. Added as well REMOTE_DEBUG_PORT to settings